app/dependencies.py

```diff
@@ -39,6 +41,24 @@ async def yoma_agent(x_api_key: str = Header(None)):
             await agent.terminate()
 
 
+async def ecosystem_or_member_agent(
```

Comment:
Not sure if there are any existing end points that require an admin version of the eco system agent or member agent. If there are an equivalent method can be added that creates the "admin" version (admin = multitenant end points added).

Can add a method cann admin_acosystem_or_admin_member_agent and then inject that via  depends where relavant

Reply:
Have added agent_selector and admin_agent_selector and full tests